tests/data/test_hubbard_structure/test_append_hubbard_parameters_parameters0_2D_xy_arsenic_.yml

```diff
@@ -0,0 +1,9 @@
+- - 0
```

Comment:
Do we need all these files ? can it be in a single yml file ?

Reply:
These are automatically generated using the `pytest` `data_regression` fixture. The main reason I added these was to:

1. also test the case where `translation` is `None`.
2. not have the test rely on the `HubbardParameters.from_tuple` method, which `append_hubbard_parameter` uses internally.

Reply:
ah ok , besides that it LGTM!